### ISSUE TITLE
plugin howto refers to "getPlugin" instead of "getPlugins"

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2727]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#2727|
|Type|defect|
|Created|2007-06-29 14:54:40Z|

<details><summary>Searchable metadata</summary>

```
trac-id__2727 2727
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1183128880000000 1183128880000000
changetime__1193392643000000 1193392643000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_1"></a>|@therve set status to `closed`|
|-|-|

(In [21442]) Fix a typo in plugin howto

Author: exarkun (almost)
Reviewer: therve
Fixes [#2727](https://github.com/twisted/twisted/issues/2727)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
